invenio_db/utils.py

```diff
@@ -78,3 +81,11 @@ def create_alembic_version_table():
         alembic.migration_context._ensure_version_table()
         for head in alembic.script_directory.revision_map._real_heads:
             alembic.migration_context.stamp(alembic.script_directory, head)
+
+
+def drop_alembic_version_table():
```

Comment:
We can do it yes but it would mean that it wouldn't be advised anymore to use sqlalchemy `create_all` and `drop_all` as they are not tested. Normally there is only one "alembic recipe" test per module and only that one need to do the extra step of dropping the alembic_version table.
Also the CLI commands `invenio db create` and `invenio db drop` are already creating and dropping the alembic_version table in addition to calling `create_all` and `drop_all`.